.github/workflows/test-backup-restore.yaml

```diff
@@ -5,42 +5,33 @@ on:
   workflow_dispatch:
     inputs:
       image:
-        description: 'GCP image for test cluster'
```

Comment:
@bastelfreak The integration testing is actually part of this PR since that is needed to make backup/restore releasable.
The quoting change is an unfortunate side-effect of auto-formatting I'm afraid...

Reply:
now this was merged with the quote changes. Now the style in the repo is very inconsistent and the diff in this PR is unnecessary big.

Reply:
Inconsistency was inevitable from this change, and will be synchronised in a separate PR.